openfe/protocols/openmm_rfe/equil_rfe_methods.py

```diff
@@ -527,10 +527,19 @@ def run(self, *, dry=False, verbose=True,
         )
 
         #  b. Write out a PDB containing the subsampled hybrid state
+        bfactors = np.zeros_like(selection_indices, dtype=float)  # solvent
```

Comment:
ultimately I think this might a bit of cleaning / expansion, but let's just go with it for now whilst we workshop exactly what we need out of all of this.